python/RESOURCE_MANAGED_BY_CLOUDFORMATION/RESOURCE_MANAGED_BY_CLOUDFORMATION.py

```diff
@@ -0,0 +1,495 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+import json
+import sys
+import datetime
+import boto3
+import botocore
+
+try:
+    import liblogging
+except ImportError:
+    pass
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::::Account'
+
+# Set to True to get the lambda to assume the Role attached on the Config Service (useful for cross-account).
+ASSUME_ROLE_MODE = False
+
+# Other parameters (no change needed)
+CONFIG_ROLE_TIMEOUT_SECONDS = 900
+
+#############
+# Main Code #
+#############
+
+allResources = dict()
+
+
+# Check if key exists in dict
+def checkKey(dict, key):
+    if key in dict.keys():
+        return True
+    else:
+        return False
+
+
+def build_cloudformation_resource_list():
+    cfn = boto3.client('cloudformation')
+    stackList = cfn.list_stacks(
+        StackStatusFilter=[
+            'CREATE_COMPLETE',
+            'ROLLBACK_COMPLETE',
+            'UPDATE_COMPLETE',
+            'UPDATE_ROLLBACK_COMPLETE',
+        ]
+    )
+    try:
```

Comment:
Leftover to be deleted?

Reply:
I was not sure if I would need to address result pagination with the CloudFormation APIs. In my tests I was able to return many resources from many stacks without pagination. For very large CloudFormation deployments, there may be a limit to how many resources can be returned in a single API call. For now I've removed this block.